src/server/game/Spells/SpellEffects.cpp

```diff
@@ -3744,33 +3744,6 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
 
                     return;
                 }
-                case 45668:                                 // Ultra-Advanced Proto-Typical Shortening Blaster
-                {
-                    if (!unitTarget || unitTarget->GetTypeId() != TYPEID_UNIT)
```

Comment:
unitTarget->GetTypeId() != TYPEID_UNIT means that the target can only be a Creature but the new code doesn't have this condition. Is that ok or should this condition be added to the new script too ?

Reply:
Spell that triggers this spell already has conditions tied to specific entries:
```sql
mysql> SELECT * FROM conditions WHERE SourceEntry=51912;
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
| SourceTypeOrReferenceId | SourceGroup | SourceEntry | SourceId | ElseGroup | ConditionTypeOrReference | ConditionTarget | ConditionValue1 | ConditionValue2 | ConditionValue3 | NegativeCondition | ErrorType | ErrorTextId | ScriptName | Comment                                                        |
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
|                      17 |           0 |       51912 |        0 |         0 |                       31 |               1 |               3 |           25434 |               0 |                 0 |         0 |           0 |            | Spell Crafty's Ultra-Advanced Proto-Typical Shortening Blaster |
|                      17 |           0 |       51912 |        0 |         1 |                       31 |               1 |               3 |           25432 |               0 |                 0 |         0 |           0 |            | Spell Crafty's Ultra-Advanced Proto-Typical Shortening Blaster |
+-------------------------+-------------+-------------+----------+-----------+--------------------------+-----------------+-----------------+-----------------+-----------------+-------------------+-----------+-------------+------------+----------------------------------------------------------------+
```